Makefile.pseudomodules

```diff
@@ -33,6 +33,7 @@ PSEUDOMODULES += lwip_stats
 PSEUDOMODULES += lwip_tcp
 PSEUDOMODULES += lwip_udp
 PSEUDOMODULES += lwip_udplite
+PSEUDOMODULES += mpu_stack_guard
```

Comment:
I considered that, but wanted to leave room for other MPUs to be used in the same way.